### PR TITLE
C/blogpost commoncrawl plugin

### PR DESCRIPTION
**Pull request by celaus**

### CODE REVIEW

File: pages/a/crate-commoncrawl.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+this gzipped, multi-line text block into a single, JSON-encoded line.
+
+### Just another InputFactory?
+
+Starting with version 0.55.0, Crate can load data from URLs, the handling of
+which is determined by an `InputFactory` class that declares the protocol it can
+handle. Instances of that class then implement a method called `getStream()` [to
+provide a stream of line-based
+JSON](https://github.com/crate/crate/blob/master/sql/src/main/java/io/crate/operation/collect/files/FileReadingCollector.java#L219)
+to the `FileReadingCollector`. This is exactly how we created our plugin.
```
Comment (bogensberger) [inline pages/a/crate-commoncrawl.html]: This is exactly what our plugin provides / This is exactly what our plugin does

File: pages/a/crate-commoncrawl.html [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+two new lines, followed by another header (starting with `WARC/1.0`). With that
+knowledge we could construct a simple parser that looks for those headers,
+sanitizes and transforms them and serializes the result into JSON. The
+`FileInput` adds glue to this, providing an unzipped stream of text to the
+parser, and returning a stream for Crate to read.
+
+### Using It
+
+Plugins integrate seamlessly with Crate. After adding the plugin to the plugins
+folder, it should be automatically loaded when Crate starts. Then a simple
```
Comment (bogensberger) [inline pages/a/crate-commoncrawl.html]: s/it should be/it is

File: pages/a/crate-commoncrawl.html [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+RESULT
+
+Whatever your questions might be, this dataset is an unlimited source of
+knowledge!
+
+## What now?
+
+Since we achieved our primary goal of importing a large amount of data, we
+decided to Open Source this project to show how easy it is to adapt Crate to
+your needs. With our implementation we hope to provide a reference to bootstrap
```
Comment (bogensberger) [inline pages/a/crate-commoncrawl.html]: With this implementation we provide another reference ...

Comment (celaus) [inline pages/a/crate-commoncrawl.html]: we are just being polite :D

File: pages/a/crate-commoncrawl.html (hunk 1 above)
Comment (celaus) [inline pages/a/crate-commoncrawl.html]: I like my version better :)

**Comment by celaus** [comment]
cc

File: pages/a/crate-commoncrawl.html [hunk 4]
```diff
@@ -0,0 +1,207 @@
+title: Plugin for Common Crawl
```
Comment (joemoe) [inline pages/a/crate-commoncrawl.html]: The title is a bit lame. What would people search for? I guess "Crate Common Crawl plugin" isn't something. I would head more into the direction "Import Data Example Plugin" or "Ingest Custom Formatted Data Plugin". Maybe even something with Copy From. I think those search terms have more values to us.

**Pull request closed without merging**